features/support/data_generator.rb

```diff
@@ -32,8 +33,8 @@ def generate_registration(business_type, operator_name = nil)
     registration_number: registration_number,
     partners: generate_partners(business_type),
     site: generate_site,
-    on_farm: true,
-    farmer: true,
+    on_farm: rand(0..1) == 1,
```

Comment:
I think a clearer way of doing this in ruby is

```suggestion
    on_farm: [true, false].sample,
```

Reply:
Looks good, thanks. I've updated this and verified the tests still work - I've pushed the change.